src/Core/src/Platform/iOS/AcceleratorExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Linq;
+using Foundation;
+using ObjCRuntime;
+using UIKit;
+
+namespace Microsoft.Maui.Platform
+{
+	internal static class AcceleratorExtensions
+	{
+		internal static UIMenuElement GetPlatformAccelerator(this IMenuFlyoutItem virtualView, IMauiContext mauiContext)
```

Comment:
Is this the correct name? `GetPlatformAccelerator`?

Reply:
I agree looks better

Reply:
yeah that sounds good to me! changing to `CreateMenuItem`